samples/bluetooth/mesh/light_dimmer/src/model_handler.c

```diff
@@ -39,6 +40,16 @@ struct dimmer_ctx {
 	struct k_work_delayable long_press_stop_work;
 };
 
+/** Context for a single scene button instance */
+struct scene_btn_ctx {
+	/** Scene client instance for this button */
+	struct bt_mesh_scene_cli client;
+	/** The currently recalled scene*/
+	uint16_t current_scene;
+	/** Time when button was pressed */
+	int64_t pressed_time;
```

Comment:
Choose to name the time variable either `pressed_time` or `press_start_time` for both button contexts.